.github/workflows/ci.yml

```diff
@@ -18,6 +18,13 @@ jobs:
       - name: Fetch git tags
         run: ./.github/fetch_to_tag.sh
 
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v3
+        with:
+          cmdline-tools-version: 9477386
```

Comment:
The runner image [ships with this version](https://github.com/actions/runner-images/blob/main/images/ubuntu/toolsets/toolset-2004.json#L79). Do you think we can avoid adding a 3rd party tool for it?

Reply:
yes, but they upgraded the build tools and platform version from 27 to 31. Ok, I'm going to download it manually

Reply:
Here is an [example](https://github.com/rollbar/rollbar-java/pull/316/files?diff=unified&w=1#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR84-R94) on how to install the old platform version with the existing SDK manager. You don't need to manually download it, or use 3rd party tools.